log4j-appender/src/test/java/org/apache/kafka/log4jappender/KafkaLog4jAppenderTest.java

```diff
@@ -190,7 +190,7 @@ private Properties getLog4jConfigWithRealProducer(boolean ignoreExceptions) {
         props.put("log4j.appender.KAFKA.layout.ConversionPattern", "%-5p: %c - %m%n");
         props.put("log4j.appender.KAFKA.BrokerList", "127.0.0.2:9093");
         props.put("log4j.appender.KAFKA.Topic", "test-topic");
-        props.put("log4j.appender.KAFKA.RequiredNumAcks", "-1");
+        props.put("log4j.appender.KAFKA.RequiredNumAcks", "1");
```

Comment:
This test also needs to be updated to match what we do in the code.

Reply:
i also took the liberty to re-revert to
```
props.put("log4j.appender.KAFKA.RequiredNumAcks", "-1");
```
in `KafkaLog4jAppenderTest.getLog4jConfig()`